xtts-v2-truss/config.yaml

```diff
@@ -0,0 +1,17 @@
+environment_variables: {}
```

Comment:
you can drop the keys you're note using (like `environment_variables`  and `external_package_dirs` and `python_version` since you're using a base image)